_includes/main.js

```diff
@@ -5,3 +5,69 @@ trigger.addEventListener('click', function(e) {
     e.preventDefault();
     element.classList.toggle('hide');
 });
+
+function candidateTable() {
+  const tableBody = document.querySelector("#tableCandidateBody");
+  const caption = document.querySelector("#candidate_table > caption");
+  let employeeCount = parseInt(document.getElementById('employee_count').value);
+  let list_owners = document.getElementById('list_owners').value
+  let list_name = document.getElementById('list_name').value || list_owners
+  let recommended_candidates_size = worksCouncilSize(employee_count)*2;
```

Comment:
```suggestion
  let recommended_candidates_size = worksCouncilSize(employeeCount)*2;
```

this fixes the function!